src/Umbraco.Web.UI.Client/src/views/webhooks/overlays/edit.controller.js

```diff
@@ -70,29 +86,29 @@
       const resource = isContent ? contentTypeResource : mediaTypeResource;
       $scope.model.contentTypes = [];
 
-      selection.forEach((entity) => {
+      selection.forEach(entity => {
         resource.getById(entity.key)
-          .then((data) => {
+          .then(data => {
             $scope.model.contentTypes.push(data);
           });
       });
     }
 
     function clearContentType(contentTypeKey) {
-      if (Array.isArray($scope.model.webhook.contentTypeKeys)) {
+      if (Utilities.isArray($scope.model.webhook.contentTypeKeys)) {
```

Comment:
Just for my own knowledge, what is the difference between using Utilities & Array ? 🤔

Reply:
Not much, it is just a wrapper we use much elsewhere in codebase and some of the methods have replaced the angular functions.
https://github.com/umbraco/Umbraco-CMS/blob/b1b48ec5cdeb29972b6dd16f58992d703d1cd70d/src/Umbraco.Web.UI.Client/src/utilities.js#L26

Reply:
![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/bd7b7b17-1fe4-4ad8-944d-7c373b2ab08b)

Reply:
Most of the time there won't be a difference.
Not sure if `Array.isArray(val)` could return a different result than `val instanceof Array`

I tried with `[]`, `""`, `null`, `undefined`, `NaN` .. all returned same result.